frontend/packages/topology/integration-tests/features/topology/resource-quota-warning.feature

```diff
@@ -23,5 +23,18 @@ Feature: Update user in topology page if Quotas has been reached in a namespace
              When user navigates to Topology page
               And user clicks on link to view resource quota details
              Then user is redirected to resource quota list page
-   
+        
+
+        @regression
+        Scenario: deployment node side-panel shows alert when resource quota is reached: T-19-TC03
+            Given user has created another workload with resource type deployment
```

Comment:
Can we include the name as well:
```suggestion
            Given user has created workload "ex-node-js1" with resource type deployment
```

Reply:
You can even use this already present step definition `user has created workload ex-node-js1"  with resource type "deployment"`